addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -349,7 +349,17 @@ def _l10n_es_tbai_get_invoice_values(self, invoice, cancel):
         if values['is_refund']:
             values['credit_note_code'] = invoice.l10n_es_tbai_refund_reason
             values['credit_note_invoice'] = invoice.reversed_entry_id
-
+        # OSS taxes
+        is_oss = False
+        company = invoice.company_id
+        country = company.account_fiscal_country_id
+        oss_tax_groups = self.env['ir.model.data'].search([
+            ('module', '=', 'l10n_eu_oss'),
+            ('model', '=', 'account.tax.group')])
+        oss_taxes = self.env['account.tax'].search([
+            ('company_id', '=', company.id),
+            ('country_id', '=', country.id),
+            ('tax_group_id', 'in', oss_tax_groups.mapped('res_id'))])
```

Comment:
we could have a common method as in 17 still l10n_es_edi_tbai depends on l10n_es_edi_sii (although probably the logic of finding the claveregimen is double)

Reply:
Indeed. I've create a common method `_has_oss_taxes`